managers/sfrElasticRecord.py

```diff
@@ -76,7 +76,7 @@ def enhanceWork(self):
         ]
         
         self.work.is_government_document = SFRElasticRecordManager.addGovDocStatus(
-            self.dbWork.measurements
+            self.work.editions.measurements
```

Comment:
`self.work.editions` is an array, so this has to become a bit more complicated unfortunately, either here or in the `addGovDocStatus` method.

Reply:
Ok. I decided to change the addGovDocStatus method.

Reply:
sorry if I was unclear! `self.work.editions` is an array of `edition` objects, each of which has a `measurements` array, so that's the complexity that the method has to account for. Should be very similar logic-wise to the script that you wrote.

Once this is fixed up the PR should be good to go!